packages/utils/upgrade/src/tasks/upgrade/requirements/common.ts

```diff
@@ -11,7 +11,7 @@ export const REQUIRE_GIT_CLEAN_REPOSITORY = requirementFactory(
 
     if (!status.isClean()) {
       throw new Error(
-        'Repository is not clean. Please commit or stash any changes before upgrading'
+        'Repository is not clean. Please consider committing or stashing any changes before upgrading'
```

Comment:
I liked the old one, it's shorter

Reply:
My thinking was that since it's optional, I didn't want the user to think it wouldn't work without it, hence the change.
But I'll trust you on this :slightly_smiling_face: